public/pages/DetectorResults/components/DetectorState/DetectorInitializing.tsx

```diff
@@ -34,14 +48,24 @@ export const DetectorInitializing = (props: DetectorInitializingProps) => {
       }
       body={
         <Fragment>
-          <p>
-            Based on your latest update to the detector configuration, the
-            detector is collecting sufficient data to generate accurate
-            real-time anomalies.
-          </p>
-          <p>
-            The longer the detector interval is, the more time this will take.
-          </p>
+          {!isInitOvertime
+            ? [
+                <p>
+                  Based on your latest update to the detector configuration, the
+                  detector is collecting sufficient data to generate accurate
+                  real-time anomalies.
+                </p>,
+                <p>
+                  The longer the detector interval is, the more time this will
+                  take.
+                </p>,
+              ]
+            : [
+                <p>
+                  {`Detector initialization is not complete because ${initErrorMessage}.`}
```

Comment:
There can be grammar error. Some of initErrorMessage has verb and some don't.  How about replace because with a colon?

Reply:
The only 2 initErrorMessages have verb:
`collected data is insufficient`: https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/159/files#diff-bf4a77e66d4a8caf187ddf3925f5d5c2R70
`no data exists in current time window`: https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/159/files#diff-bf4a77e66d4a8caf187ddf3925f5d5c2R76

Also, I am waiting for response from Ashwin on the wording. I will change it later after he reviews it.

Reply:
I see.  I looked at public/pages/DetectorResults/utils/constants.ts.  You can leave it as it is.